locales/en-US/editor.json

```diff
@@ -1,7 +1,7 @@
 {
     "workspace": {
         "label": {
-            "name": "Name"
+            "name": "Name:"
```

Comment:
It was used originally on line 146 of `index.mst`, but the colon was still hardcoded. I brought it into the NLS string since some languages handle colons different (French, for instance, has a space between a colon and the word it follows). I checked that it wasn't otherwise used in the editor code by doing a find on the built `red.js` in the `public` folder. Ideally an NLS string should only be used in one place.